ui/js/controllers/tcjobactions.js

```diff
@@ -1,4 +1,5 @@
 "use strict";
+var jsonSchemaDefaults = require('json-schema-defaults');
```

Comment:
Drive by: I think it would be better to put this inside the controller function, below (to avoid polluting the global scope)